drivers/bus/acpi/main.c

```diff
@@ -311,6 +311,11 @@ ACPIDispatchDeviceControl(
 
               status = STATUS_PENDING;
               break;
+               
+           // FIXME: When CORE-14134 gets fixed, IOCTL_BATTERY_QUERY_TAG case should be removed
+           case IOCTL_BATTERY_QUERY_TAG:
+              DPRINT("Unsupported IOCTL with ID 294040.\n");
```

Comment:
Indicating IOCTL_BATTERY_QUERY_TAG by name instead of ControlCode would be more useful.

Reply:
@ThFabba Like Unsupported IOCTL_BATTERY_QUERY_TAG inside DPRINT() you mean? I'm actually confused.